services/web/server/src/simcore_service_webserver/login/utils.py

```diff
@@ -125,17 +125,22 @@ async def send_mail(app: web.Application, msg: MIMEText):
         hostname=cfg.SMTP_HOST,
         port=cfg.SMTP_PORT,
         use_tls=cfg.SMTP_TLS_ENABLED,
+        start_tls=cfg.SMTP_STARTTLS_ENABLED,
     )
     log.debug("Sending email with smtp configuration: %s", pformat(smtp_args))
     if cfg.SMTP_PORT == 587:
         # NOTE: aiosmtplib does not handle port 587 correctly
         # plaintext first, then use starttls
         # this is a workaround
         smtp = aiosmtplib.SMTP(**smtp_args)
-        await smtp.connect(use_tls=False, port=cfg.SMTP_PORT)
-        if cfg.SMTP_TLS_ENABLED:
-            log.info("Starting TLS ...")
+        if cfg.SMTP_STARTTLS_ENABLED:
+            log.info("Unencrypted connection attempt to mailserver ...")
+            await smtp.connect(use_tls=False, port=cfg.SMTP_PORT)
+            log.info("Starting STARTTLS ...")
+            log.warning("Certificates are not validated for STARTTLS on port 587!")
```

Comment:
not sure I understand the reason for this warning here

Reply:
It is not really perfect to use encrypted communication but trust all certificates, the gain in security is marginal. I just thought the least I can do at this point (didnt wanna open up a whole big pandoras box and follow up on why it is coded like this now), was to add a warning.

But of course this log can be nerfed or removed

Reply:
I would suggest something like the following:

```python
log.warning(
    f"Certificates are not validated for STARTTLS (as it happens for TLS) on port {cfg.SMTP_PORT}!"
)
```

Reply:
I am not sure I understand the whole thing here. Is this a warning that no one will check even though we should tackle it? in which case please create an issue.
Is this workaround here because of the library?  Is that the way it works? then I would not show any warning.
And if this is a security issue, should we really use starttls?

Reply:
It's slightly better than unencrypted. In theory we should get a better mail server, since we run it, which supports TLS and be done with it.